scripts/question_answering/index.rst

```diff
@@ -0,0 +1,72 @@
++                   +------------------+---------------+---------------+--------------------------------------------+--------------------------------------------+--------------+
+|                   |  Attention heads |       1       |       2       |                      1                     |                      8                     |       8      |
++                   +------------------+---------------+---------------+--------------------------------------------+--------------------------------------------+--------------+
+|                   |      Blocks      |       7       |       7       |                      7                     |                      7                     |       7      |
++                   +------------------+---------------+---------------+--------------------------------------------+--------------------------------------------+--------------+
+|                   |    Conv layers   |       2       |       2       |                      2                     |                      2                     |       2      |
++-------------------+------------------+---------------+---------------+--------------------------------------------+--------------------------------------------+--------------+
+|                   |        F1        |      79.0     |      79.5     |                    79.8                    |                    80.1                    |     82.7     |
++                   +------------------+---------------+---------------+--------------------------------------------+--------------------------------------------+--------------+
+|                   |        EM        |      69.8     |      70.2     |                    70.7                    |                    70.8                    |     73.6     |
++                   +------------------+---------------+---------------+--------------------------------------------+--------------------------------------------+--------------+
+|                   |      Command     |      [1]      |      [2]      | `[3] <https://github.com/NLPLearn/QANet>`_ | `[4] <https://github.com/NLPLearn/QANet>`_ |       /      |
++-------------------+------------------+---------------+---------------+--------------------------------------------+--------------------------------------------+--------------+
+
+
+Usage
+~~~~~~~~~~~~~~~~~~~~
+To preprocess the data, you need to download `dataset <https://rajpurkar.github.io/SQuAD-explorer/>`_ and `glove <https://nlp.stanford.edu/projects/glove/>`_ pretrained word vector.
+And put all three files in the same directory with data_process.py and run the following code: 
+.. code-block:: console
+
+    $ python data_process.py
+
+[1] QANet_MXNet_1 (Val set F1= 79.0 EM= 69.8)
+
+Remember to modify the hyperparameter in config.py file to the value listed in the above table.

```

Comment:
Having preset config is great. On the other hand, instead of asking users to update the config file, it might be better to provide the presets beforehand so that users can switch with, say, a string argument to the script.

Reply:
Done.